lib/healthchecks/version_check.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module HealthChecks
+  class VersionCheck < OkComputer::Check
+    def check
+      version = ENV.fetch('APP_VERSION', nil)
```

Comment:
What about `version = ENV.fetch('APP_VERSION', 'unknown')` and that way there's always something to display? Or is it important that a missing version key be communicated as a failure?

Reply:
I don't think it's important. "unknown" sounds good to me!